app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionBlockTooltip/serviceVersionsBlock.scss

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.versions-list {
+  font-size: 11px;
+  color: $COLOR--charcoal-grey;
+}
+
+.deprecated-icon {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  background: url('../../../img/info-small-deprecated.svg') center no-repeat;
```

Comment:
You can move this icon closer to component if you use it only here. If you use it in mane places move it to the common icons folder. Also you can use Parser to set icon directly in jsx
You repeat yourself(.current-icon, .deprecated-icon).  Pls read comment above you don't needed at all